src/DynamoRevit/Models/RevitDynamoModel.cs

```diff
@@ -611,7 +617,7 @@ private void OnResetMarkNodesAsDirty(bool markNodesAsDirty)
                 workspace.ResetEngine(EngineController, markNodesAsDirty);
         }
 
-        public void SetRunEnabledBasedOnContext(View newView)
+        public void SetRunEnabledBasedOnContext(View newView, bool doContextAvailable)
```

Comment:
last thought: can you make this either DOContextAvailable or DocumentObject.... I think the intended pursue  will just be more clear.